packages/server/index.d.ts

```diff
@@ -13,6 +13,8 @@
 /// <reference path="../../cli/types/cypress-global-vars.d.ts" />
 /// <reference path="../../cli/types/cypress-type-helpers.d.ts" />
 
```

Comment:
nit extra space here:

```suggestion
```

Reply:
I think that was intentional to separate the cli types from the driver types.